crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -860,6 +860,11 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
     ///
     /// See [`World::despawn`] for more details.
     ///
+    /// # Note
```

Comment:
Is it possible to mention the alternatives in `bevy_hierarchy`? I know they can't be directly linked to unfortunately, but noting some kind of usable alternative is better than having people needing to ask.

Reply:
Sure, updated. I'm not sure how to test that the links to `DespawnRecursive` and `DespawnDescendants` actually work.

Reply:
Ok the links didn't work, updated with no links now.